2.train-VAE/betavae.py

```diff
@@ -24,9 +24,7 @@ def __init__(self, input_dim, latent_dim, beta):
             nn.ReLU(),
         )
         self.decoder = nn.Sequential(
-            nn.Linear(latent_dim, input_dim), 
-            nn.BatchNorm1d(input_dim), 
-            nn.Sigmoid()
+            nn.Linear(latent_dim, input_dim), nn.BatchNorm1d(input_dim), nn.Sigmoid()
```

Comment:
this is what happens after `black`?

Consider implementing [pre-commit](https://pre-commit.com/) in your workflow :) I think it could help a bunch with productivity by emphasizing more of your focus on ML/data science and decreasing attention away from formatting

Reply:
Worked on configuring black to not edit lines like this!